src/BangBang.jl

```diff
@@ -61,6 +62,10 @@ function unique!! end
 include("NoBang/NoBang.jl")
 using .NoBang: Empty, SingletonVector, singletonof
 
+@static if !isdefined(Base, :get_extension)
+    include("../ext/BangBangTablesExt.jl")
+end
```

Comment:
So I think we want to turn this into
```suggestion
@static if !isdefined(Base, :get_extension)
    using Tables
    include("../ext/BangBangTablesExt.jl")
end
```
because previously it was unconditionally loaded.

Reply:
Maybe it's best to treat Tables.jl like the other weak deps and use it with Requires.jl on old versions?

Reply:
Isn't that a breaking change? At the start I had it in that way, before knowing that Tables was loaded inconditionally

Reply:
Mhm right. I guess let's just make it not an extension and put in a note saying that next time we make a breaking release we'll move it to an extension?